ui.apps/src/main/content/jcr_root/etc/notification/email/hclecomm/store-pick-up-email-template.html

```diff
@@ -1,7 +1,7 @@
 From:deepalia@hcl.com
```

Comment:
@anchal-a : can u pl chnage this to email to no-reply@hcl-ecommerce.ccm

Reply:
Done.